contracts/governance/IGovernor.sol

```diff
@@ -301,7 +309,8 @@ interface IGovernor is IERC165, IERC6372 {
 
     /**
      * @notice module:user-config
-     * @dev Minimum number of cast voted required for a proposal to be successful.
+     * @dev Minimum number of cast voted required for a proposal to be successful. Both FOR and ABSTAIN votes are counted
+     * towards the quorum.
```

Comment:
There are counting modules out there that use ranking systems. In there case its all handled differently and there is no FOR/ABSTAIN vote

Reply:
removed